packages/connect-node-test/src/node-readme.spec.ts

```diff
@@ -1,288 +0,0 @@
-// Copyright 2021-2024 The Connect Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-import * as http2 from "http2";
-import { Message, MethodKind, proto3 } from "@bufbuild/protobuf";
-import type { PartialMessage } from "@bufbuild/protobuf";
-import {
-  createContextKey,
-  createContextValues,
-  createPromiseClient,
-  createRouterTransport,
-} from "@connectrpc/connect";
-import type { ConnectRouter } from "@connectrpc/connect";
-import {
-  connectNodeAdapter,
-  createGrpcTransport,
-  createGrpcWebTransport,
-  createConnectTransport,
-} from "@connectrpc/connect-node";
-import { createWritableIterable } from "@connectrpc/connect/protocol";
-
-/* eslint-disable @typescript-eslint/require-await */
-
-describe("node readme", function () {
```

Comment:
Looks like this file grew to be a mixed bag of tests. It started out as test coverage for the examples we give in the package readme, and I think it's a good thing to retain.

If your concern is CI time, I think it would be better to switch to Jest and run tests in parallel. The Karma & Jasmine combo for browserstack will need to be replaced at some point anyways.